app/views/enterprises/shop.html.haml

```diff
@@ -5,7 +5,7 @@
 - content_for(:image) do
   = current_distributor.logo.url
 
-= render partial: 'json/injection_ams', locals: enterprises
+= inject_enterprise_shopfront(@enterprise)
```

Comment:
it'd be really good to stick to locals instead of implicit ivars. It's a lesson learned in the Rails community: the implicitness of ivars becomes soon a mess.